activesupport/lib/active_support/cache/mem_cache_store.rb

```diff
@@ -192,7 +192,7 @@ def read_entry(key, options)
 
           def write_entry(key, entry, options) # :nodoc:
             retval = super
```

Comment:
we dont need to call super and write to the local_cache i guess.
We can remove the retval var and just call super in the last line of the method. that would be more consistent with https://github.com/rails/rails/blob/5217db2a7acb80b18475709018088535bdec6d30/activesupport/lib/active_support/cache/strategy/local_cache.rb#L116-L119 too